ui/shared/status/StatusRadioButtonRow.qml

```diff
@@ -40,18 +51,13 @@ Rectangle {
         rightPadding: 0
         checked: root.checked
         onCheckedChanged: root.radioCheckedChanged(checked)
-    }
-
-    MouseArea {
-        cursorShape: Qt.PointingHandCursor
-        anchors.fill: parent
-        hoverEnabled: true
-        onEntered: root.isHovered = true
-        onExited: root.isHovered = false
-        onClicked: {
-            if (!radioButton.checked) {
-                root.radioCheckedChanged(true)
-            }
+        MouseArea {
+            hoverEnabled: true
+            cursorShape: Qt.PointingHandCursor
+            anchors.fill: parent
+            onPressed: mouse.accepted = false
+            onEntered: root.isHovered = true
```

Comment:
Should you have an `onExited` as well?

Reply:
It's not needed here because when we exit the button, we're still inside the rectangle. So exiting the rectangle will do the job